can/tests/test_parser_speed.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import glob
+import os
+import random
+import time
+import unittest
+
+from opendbc import DBC_PATH
+from opendbc.can.packer import CANPacker
+from opendbc.can.parser import CANParser
+from opendbc.can.tests.test_packer_parser import can_list_to_can_capnp
+from opendbc.generator.generator import generated_suffix
+
+
+class TestCANParserSpeed(unittest.TestCase):
```

Comment:
just call this TestCANParser. speed is just one test case. packer + parser combo tests can remain separate